src/html_support_files/odoc.css

```
@@ -95,7 +95,10 @@
 
 :root,
 .light:root {
-   --main-background: #FFFFFF;
+
+  scroll-padding-top: 100px;
```

Shouldn't this be in `em`, like the font size and the margin of items ?

---

More than a specific unit, shouldn't this be computed from `--search-bar-height` and `--search-padding-top` to be sure it is enough? Something like:

```suggestion
  scroll-padding-top: calc(var(--search-bar-height) + var(--search-padding-top) + 20px);
```
Where `20px` can be replaced by a size in `em` if needed.